### PR TITLE
Remove redundant steps & improve README.md

### PR DESCRIPTION
**Pull request by ethanzrd**
The installation script also installs portaudio as part of the Conda environment to avoid having the user install it on their device, eliminating the first step.
The README file addresses troubleshooting and initial configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,16 +30,17 @@ scoop install ffmpeg
 - Beam Size: Adjust the amount of transcriptions generated and considered, which affects accuracy and transcription generation time.
 - Transcription Method: Choose "real-time" for real-time diarization and transcriptions, or "sequential" for periodic transcriptions with more context.
 
-## Latency
-If you want minimal latency, use the real-time mode. If you don't mind growing latency and prioritize accuracy, use the sequential mode.
+## Troubleshooting
+
+- If you're unable to connect from the client to the server, use an ngrok tunnel to expose port 8000.
```
Comment (saharmor) [inline README.md:35]: Why would this happen?

File: README.md [hunk 2]
```diff
@@ -49,16 +30,17 @@ scoop install ffmpeg
...
 
-1. You may have ffmpeg errors when locally running the application. Create an issue including your operating system and the steps to reproduce.
-2. You may encounter problems when building the wheel for "safetensors". On MacOS, run `brew install rust` to fix it.
-3. In the sequential mode, the application may crash if the first transcription batch contains no speech.
-4. In the sequential mode, there may be uncontrolled speaker swapping, which can be fixed by using pyannote's building blocks and handling speakers manually.
-5. In real-time mode, audio data not meeting the transcription timeout won't be transcribed.
+1. On MacOS, there's a clash between av files preventing transcription (works well on Google Colab with Python 3.8).
+2. In the sequential mode, there may be uncontrolled speaker swapping, which can be fixed by using pyannote's building blocks and handling speakers manually.
+3. In real-time mode, audio data not meeting the transcription timeout won't be transcribed.
+4. Speechless batches will cause errors.
```
Comment (saharmor) [inline README.md:43]: Add a link to the issue you've opened

File: backend/config.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 from diart import PipelineConfig
 from enum import Enum
 
+TRANSCRIPTION_DEVICE = "cuda"  # use 'cpu' if it doesn't work
+COMPUTE_TYPE = "int8_float16"  # use float32 with cpu
```
Comment (saharmor) [inline backend/config.py:5]: Maybe I'm missing something, but the comment seems off. Float 32 though using `int8_float16`

**Review by saharmor — APPROVED** [review]

**Comment by saharmor** [comment]
@ethanzrd I approved and merged but please reply to my comments or just go ahead and fix them if relevant

File: README.md (hunk 1 above)
Comment (ethanzrd) [reply README.md:35]: Honestly, no idea. Worked fine on MacOS, but didn't work on Windows. If I had to take a guess, I'd say the connection is blocked.

File: backend/config.py (hunk 3 above)
Comment (ethanzrd) [reply backend/config.py:5]: If one is using "cpu" as their transcription device, it should be float32. float16 wouldn't make that much of a difference even if supported.

File: README.md (hunk 1 above)
Comment (Epresin) [reply README.md:35]: > Honestly, no idea. Worked fine on MacOS, but didn't work on Windows. If I had to take a guess, I'd say the connection is blocked.

After changing http://0.0.0.0:8000/ to http://localhost:8000/ in App.js, it is able to run.

Comment (ethanzrd) [reply README.md:35]: @Epresin Good find! Appreciate it :)
Seems to work just fine on MacOS as well, so it's probably the safer bet :)

**Pull request merged**